centreon/packages/ui/src/Dialog/index.tsx

```diff
@@ -93,14 +106,20 @@ const Dialog = ({
       )}
       <DialogActions className={dialogActionsClassName}>
         {onCancel && (
-          <Button color="primary" disabled={cancelDisabled} onClick={onCancel}>
+          <Button
+            color="primary"
+            data-testid={dataTestIdCanceledButton}
```

Comment:
is this actually necessary ? in the scope of the dialog.data-testid these buttons are unique

what about a generic id for the buttons ?
```
data-testid={'cancel'}
data-testid={'confirm'}
```

Reply:
emm there is cancel notification creation and cancel notification update , but i'll ask PO if it s necessary to track if users cancel a notification creation/update 🤔

Reply:
selectors like these should be possible, no?
```
div[data-testid="create-notification"] button[data-testid="confirm"]
div[data-testid="create-notification"] button[data-testid="cancel"]

div[data-testid="update-notification"] button[data-testid="confirm"]
div[data-testid="update-notification"] button[data-testid="cancel"]
```
everything can be tracked, even with generic ones

Reply:
🤔  for me i just pass the data-testid through the button , that should describe the all action , i don t use the selectors , maybe how they tracks the users they use the selectors 🤔  , the ticket describe that i should pass one data-testId for each button describing the action